config/routes.js

```diff
@@ -0,0 +1,62 @@
+const bcrypt = require ('bcryptjs');
+const db = require('../data/dbConfig.js');
+const { authenticate, generateToken } = require('../auth/authenticate.js');
+
+module.exports = server => {
+    server.post('/api/register', register);
+    server.post('/api/login', login);
+}
+
+function register(req, res) {
+    // implement user registration
+    const user = req.body;
+    //console.log('1. req.body:', user)
+    if(user.username && user.password) {
+      const hash = bcrypt.hashSync(user.password, 3)
```

Comment:
While it is very quick, three iterations seems very low for a password hash.

Reply:
Sure, I was thinking to increase that number after or before the launch of the project